### PR TITLE
Bump solr.max.booleanClauses 1024 to 8192

### PR DESCRIPTION
**Pull request by thomasegense**
solr.max.booleanClauses  changed in solr.config from 1024 to 8192

pomxml+schema.xml solr config bumped to 1.7.2

### CODE REVIEW

File: src/main/solr/dssolr/conf/schema.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@ When adding fields to this schema please have the following guidelines in mind:
 1.7.0: Add warming of start-up query.
 1.7.1: Rename field migrated_from to originates_from and added which contains the word Preservica as default.
```
Comment (VictorHarbo) [inline src/main/solr/dssolr/conf/schema.xml:51]: Please add a line of documentation on what is new in 1.7.2

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,10 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added 
+- solr maxBooleanClauses increased from 1024 til 8192. This is because licensmodule queries can have several 100 terms. (solr.config=1.7.2)
+  For this to also work on a solr cloud setup the solr start script must have:  export SOLR_OPTS="-Dsolr.allowPaths=/solr-c-backup -Dsolr.max.booleanClauses=8192
```
Comment (VictorHarbo) [inline CHANGELOG.md:11]: Could and should you add this to the cloud_start.sh script in the bin folder of the project, which contains a solr cloud setup?

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Minor comments

File: CHANGELOG.md (hunk 2 above)
Comment (thomasegense) [reply CHANGELOG.md:11]: Added to the custom cloud_start.sh script

**Review by VictorHarbo — APPROVED** [review]
Looks good

**Pull request merged**